data/json/items/comestibles.json

```diff
@@ -14680,8 +14705,8 @@
         "tool" : "null",
         "volume" : 1,
         "charges" : 3,
-        "bashing" : 1,
-        "fun" : 6
+        "flags" : ["EATEN_HOT"],
+        "fun" : 5
```

Comment:
Doesn't need a nerf. They have 2 allergens, aren't all that common and don't grant too much nutrition.